src/main/resources/templates/fragments/header.html

```diff
@@ -5,7 +5,6 @@
 </head>
 <body class="govuk-template__body">
 <th:block th:fragment="header">
-  <a class="govuk-skip-link" href="#main-content">Skip to main content</a>
```

Comment:
https://www.w3.org/TR/WCAG20-TECHS/G1.html
This is reccommended by W3C, if we're removing it then we ought to replace it with something

Reply:
same

Reply:
I did, it's in the baseLayout

Reply:
Do you mean that hard coded thing that is pointing at a .css?

Reply:
No, the other change to the base layout

Reply:
Sorry, I forgot that it wasn't the only change. Deleted the other comment